e2e/testdata/fn-render/fnresult-fn-success/.expected/config.yaml

```diff
@@ -19,5 +19,3 @@ exitCode: 0
 # because the results produced by search replace function gets changed
 # in the subsequent runs. So skipping it for now.
 nonIdempotent: true
```

Comment:
@droot Why do we have idempotent packages? We should remove this option all-together.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/2193

Reply:
Good question. This test-case tests function result for success case and uses `search-replace` function. Now `search-replace` output for first run is different from subsequent run because once the replacement is done, no output.

If we have a function that always produces same structured result, we don't need this test to be non-idempotent. Probably, it's easier to write a test or demo function to achieve that.  I believe this is the only test-case that runs into this problem.

Reply:
Idempotency only applies on changes made to the filesystem. `search-replace` should be idempotent in that regard. You may want to relax what the test harness asserts.